process/interceptors/processor/miniblockInterceptorProcessor.go

```diff
@@ -78,6 +83,8 @@ func (mip *MiniblockInterceptorProcessor) Save(data process.InterceptedData, _ c
 		return err
 	}
 
+	go mip.notify(miniblock, interceptedMiniblock.Hash(), topic)
```

Comment:
This will notify even the cross miniblocks dest me and even if they are not whitelisted. It is ok?

Reply:
The miniblocks "destination me" can be ignored, on subscriber (this is done on delayedBlockBroadcaster, as it waits for specific miniblocks on specific topics).

The whitelisting is done on the "cross shard destination me" miniblocks, whereas for delayed broadcaster the "cross shard source me" are relevant. 

I agree that if the miniblock is received before the metablock  notarizing the parent header then it could be ignored on destination, so waiting for suggestions here.